applications/nrf_desktop/doc/ble_bond.rst

```diff
@@ -68,7 +68,7 @@ The following diagram shows states and transitions between these states after th
 .. figure:: /images/nrf_desktop_ble_bond.svg
    :alt: nRF Desktop Bluetooth LE bond module state diagram
 
-   nRF Desktop Bluetooth LE bond module state diagram (click to enlarge)
+   nRF Desktop Bluetooth LE bond module state diagram
```

Comment:
Figure does not enlarge when it is clicked on, so removed that part of the caption for now. 
Or should this be addressed in another way?

Reply:
No, removing this makes sense.

Reply:
Leaving that out should be fine.